pyproject.toml

```diff
@@ -5,9 +5,9 @@ description = "aisee is a Python package for image classification using PyTorch
 authors = ["IIC"]
 license = "MIT"
 readme = "README.md"
-homepage = "https://aisee.readthedocs.io"
+homepage = "https://iiconocimiento.github.io/aisee/stable/"
```

Comment:
https://iiconocimiento.github.io redirects to https://iiconocimiento.github.io/aisee/stable/??

Reply:
Yes, by default it redirects to the stable documentation. This is necessary because there has to be a default documentation when you have several directories in the `gh-pages` branch.